hs_core/templates/pages/genericresource.html

```diff
@@ -835,6 +837,26 @@ <h3>Resource Specific</h3>
       const SHORT_ID = "{{ cm.short_id }}";
       const STATIC_URL = "{{ STATIC_URL }}";
       const USERS_JSON = {{ users_json|safe }};
+      const AUTHORS = [
+          {% for author in creators %}
```

Comment:
Is it any cleaner to perform this in the python file that passes `creators` to the html template?

Reply:
It's technically being done that way now, is just that the script tag exists inside the HTML file. But passing these data directly in the HTML would involve creating hidden components to attach this to. That would be way more inefficient and messy.